tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,14 +87,27 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        var stdout = wsk.cli(Seq("property", "get", "--auth")).stdout
+        stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--apihost")).stdout
+        stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--namespace")).stdout
+        stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+    }
+
     it should "show api build version" in {
+        wsk.cli(wskprops.overrides ++ Seq("property", "set"))
```

Comment:
The `show api build version` and `show api build number` tests only need valid `apihost` and `apiversion` values to be configured in the properties file.  Both those values are set via this wsk.cli() call.  Note that the default `--apiversion` value is actually the same value as the `--apiversion` value being set, so explicitly setting that value is not required.